.github/workflows/wheels.yml

```diff
@@ -83,13 +83,24 @@ jobs:
       - build_wheels
     # only run if the commit is tagged...
     if: startsWith(github.ref, 'refs/tags/')
+    environment:
+      name: publish-to-pypi
+      url: https://pypi.org/p/fontTools
+    permissions:
+      id-token: write # IMPORTANT: mandatory for trusted publishing
+
     steps:
+    - uses: actions/checkout@v5
     - uses: actions/download-artifact@v4
       with:
         # so that all artifacts are downloaded in the same directory specified by 'path'
         merge-multiple: true
         path: dist
-    - uses: pypa/gh-action-pypi-publish@v1.13.0
```

Comment:
but the official pypa/gh-action-pypi-publish should support trusted publishing, no? I don't think we _need_ to use uv publish